app/src/ui/repositories-list/repository-list-item.tsx

```diff
@@ -144,9 +156,37 @@ export class RepositoryListItem extends React.Component<
         action: this.removeRepository,
       },
     ]
+
     showContextualMenu(items)
   }
 
+  private buildAliasMenuItems(): ReadonlyArray<IMenuItem> {
+    const repository = this.props.repository
+
+    if (!(repository instanceof Repository) || !enableRepositoryAliases()) {
```

Comment:
Just curious.. is there a time that it is not an instance of a Repository?

Reply:
Hmmm.. just realized there is a type of Repositoryish. Which made me laugh... But I see that evidently it can be cloning one.